toss.py

```diff
@@ -42,59 +44,98 @@ def load_udp(args, initial_condition, lower_bounds, upper_bounds):
         args.algorithm.evaluation_stopping_criterion, 
         args.algorithm.focus_parameter, 
         args.algorithm.memory_parameter)
+    
+    # Define the UDA.
+    uda.set_bfe(bfe)
+    algo = pg.algorithm(uda)
+    
+    return algo
+
 
-    # Setup BFE machinery
+def run_optimization(args, initial_conditions, lower_bounds, upper_bounds):
```

Comment:
Fixed! The Initial condition array (although perhaps with a bit cryptic name) was meant to be a vector corresponding to given information about the initial state. For instance, it could be a vector of initial position and/or initial velocity defined by the user. In that case, it would be recognized in the optimization process and accounted for by also removing initial position/velocity from the chromosome. I defined it such that the user could choose to either optimize the initial state or letting it be fixed to any desired values.